app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/market_building.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self.beerCost = beerCost
 
     def sell(self):
+        self.buildLocation = None
```

Comment:
Doesn't it stay on the board? Or is that not what this means?

Reply:
Yeah I have no idea why that was added